changelogs/unreleased/gh-10636-json-schema.md

```diff
@@ -0,0 +1,5 @@
+## feature/schema
```

Comment:
Should be `## feature/config` or, at least, `## feature/config/schema`. See 8a0013eaeff46fa50e6acc333eb13a2cd4aa6303 for example.

Reply:
Replaced with `feature/config/schema`.